### ISSUE TITLE
High CPU load

### ISSUE THREAD

**Issue by FM1337** [kelseyhightower/nocode#762]
For some reason I'm getting unusually high CPU usage with nocode, it's causing my CPU temps to rise to about 90C. Any possible way to fix this?

**Comment by Anduin2017** [comment]
Buy a new computer.

**Comment by FM1337** [comment]
This is a $3000 computer, it should have no problem running nocode.

**Comment by QuakePhil** [comment]
Reinstall computer to defragment nocode

**Comment by FM1337** [comment]
I'll try a different flavor, I think nocode isn't compatible with arch

**Comment by FM1337** [comment]
UPDATE:

Got it running on my raspberry pi 3 toaster. So I think it's an issue with Arch

**Comment by CXuesong** [comment]
You need noCPU acceleration to run `nocode`.

**Comment by vassilvk** [comment]
@FM1337, I just ran `nocode` on my odroid-c2 cluster. So far so good - all nodes running steady at 45C.

**Comment by FM1337** [comment]
@vassilvk yeah it looks like a problem with Arch, the kernel needs to have noCPU accleration support added before it will work correctly.

**Comment by LevFlavien** [comment]
you just need some more ram https://downloadmoreram.com/

**Comment by eddynetweb** [comment]
Have you tried turning it on and off?

**Comment by adrianoresende** [comment]
`rm -rf /`
